README.md

```diff
@@ -10,7 +10,7 @@ annotating test files.
   r'''
 const TestClass1NameLength = 10;
 
-const TestClass1NameLowerCase = testclass1;
+const TestClass1NameLowerCase = 'testclass1';
```

Comment:
Since the the generators' output can now end up in Dart files, the output should contain no Dart errors to not alarm the linter. So generators where were changed to produce code with no errors.

README is not runnable but it's better to align as well.